app/shared/src/services/Translate.js

```diff
@@ -0,0 +1,7 @@
+// @flow
```

Comment:
For the future reference: I would prefer to think about better structure instead of adding everything to the `services`. It's like a bag for everything. I think something like `shared/translations` or even new package for translations is a little bit better approach. What do you think?

Reply:
I thank that makes more sense, I will implement that structure

Reply:
Now or in the future? Should I merge it or wait?

Reply:
I will do it now for translations